DougieDev/whats-cookin-dougie-rachel
# Rw icon click functionality

#### What changes/additions are a part of this PR?
-I tackled 2 issues here, as I realized that in attempting one, I needed to do the other first:
- I added a function that instantiates a random User on page load, and displays their name in the H2 of the page
- I added a few functions for when the heart icon & notebook icon are clicked. When this happens, the recipe is added/removed to the user's favorite recipes array (heart icon) or meals to cook array (notebook icon) & the icon also toggles between active/inactive state
#### Any background context you want to provide?
Website functionality is really getting there! 
#### Where should the reviewer start?
Scripts.js houses most of these updates, starting around line 28
#### How should this be tested (test suite/manually)?
 Add in console.log statements to see how the arrays of user change when clicking the icons on each recipe card. If you pull down the branch, you can also open the index.html file to see the DOM changes in action 
#### What are the relevant tickets?
Tickets #32 & #36 
#### Screenshots (if helpful)
#### Any questions/additional comments:
